### DISCUSSION TITLE
Welcome to superfree-binary-bot Discussions!

### DISCUSSION THREAD

**Discussion by alanvito1** [alanvito1/ORSTAC#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Nique99** [comment]
How can I create my bot

**Reply by alanvito1** [reply]
here you have several models that you can see them working and replicate strategies at your discretion

**Reply by alanvito1** [reply]
Used to twitter haha

**Reply by alanvito1** [reply]
Let's do it
You log in into Deriv dBot Plataform and load any bot here

**Reply by alanvito1** [reply]
You can download all by a computer desktop if you have a mobile try put it in desktop mode

**Reply by alanvito1** [reply]
Once you understand the logic of built robots you can generate your own strategy
